config/env/development.js

```diff
@@ -39,5 +39,8 @@ module.exports = {
 				pass: process.env.MAILER_PASSWORD || 'MAILER_PASSWORD'
 			}
 		}
-	}
+	},
+    mongoose: {
+        debug: true
```

Comment:
I think we should make it `db: {}` and put the db key with the url under this object as url.